webrender_traits/src/types.rs

```diff
@@ -397,13 +397,13 @@ pub struct ScrollLayerId {
 #[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub enum ScrollLayerInfo {
     Fixed,
-    Scrollable(usize)
+    Scrollable(usize, ServoScrollRootId)
```

Comment:
Hrm. I'm not sure. I kept the same naming scheme as ServoStackingContextId in the off-chance that WebRender also grew an understanding (and an id) for ScrollRoots. I guess we can land this PR and then investigate later once my first round of work is complete.